renovate.json

```diff
@@ -20,6 +20,63 @@
       ],
       "groupName": "All npm dependencies",
       "groupSlug": "npm-dependencies"
+    },
+    {
+      "enabled": false,
+      "matchFileNames": [
+        "src/content/docs/structure/packages.md"
+      ]
+    }
+  ],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      "matchStrings": [
+        "repository:\\s*(?:(?:oci://)?(?<registry>ghcr\\.io)/(?<depPath>[^\\s]+))[\\s\\S]*?ref:\\s*(?<currentValue>[^\\s\"']+)"
+      ],
+      "datasourceTemplate": "docker",
+      "depNameTemplate": "{{registry}}/{{depPath}}",
+      "versioningTemplate": "regex:^v?(?<major>\\d+)(\\.(?<minor>\\d+))?(\\.(?<patch>\\d+))?",
+      "extractVersionTemplate": "^(?<version>v?\\d+\\.\\d+(?:\\.\\d+)?)(?<suffix>-uds\\.\\d+(?:-[\\w.-]+)?)?$",
+      "autoReplaceStringTemplate": "{{{newValue}}}{{{suffix}}}",
+      "registryUrlTemplate": "https://{{registry}}"
+    },
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      "matchStrings": [
+        "(?m)^\\s*-\\s+['\"]?(?<depName>(?:ghcr\\.io|quay\\.io|docker\\.io)/[^\\s:@'\"]+(?:/[^\\s:@'\"]+)*)['\"]?:(?<currentValue>[\\w.\\-]+)\\s*$"
+      ],
+      "datasourceTemplate": "docker",
+      "versioningTemplate": "docker"
+    },
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      "matchStrings": [
+        "(?m)name:\\s*[^\\n]+[\\s\\S]*?version:\\s*(?<currentValue>\\d+\\.\\d+\\.\\d+)[\\s\\S]*?url:\\s*[\"']?(?<gitUrl>https?://(?:github\\.com|gitlab\\.com|bitbucket\\.org)/[^\\s\"']+?\\.git)[\"']?\\s*$"
+      ],
+      "datasourceTemplate": "git-tags",
+      "depNameTemplate": "{{gitUrl}}",
+      "versioningTemplate": "semver"
+    },
+    {
+      "customType": "regex",
+      "managerFilePatterns": [
+        "/^src\\/content\\/docs\\/.*\\.(md|mdx)$/"
+      ],
+      "matchStrings": [
+        "https://raw\\.githubusercontent\\.com/(?<depName>[^/]+/[^/]+)/(?<currentValue>v?\\d+\\.\\d+\\.\\d+)/[^\\s\"')]+"
+      ],
+      "datasourceTemplate": "github-tags",
+      "versioningTemplate": "semver-coerced"
     }
```

Comment:
Comparing this one to the [uds-common match](https://github.com/defenseunicorns/uds-common/blob/v1.21.5/config/renovate.json5#L63-L75) it seems like this drops `/refs/tag`. I think that both are supported by maru so might not matter (mostly trying to parse these for my own understanding and notes).